larpix/format/message_format.py

```diff
@@ -95,3 +82,53 @@ def dataserver_message_encode(packets, key_parser=None, version=(1,0)):
             msg += packet.bytes() + b'\x00'
         msgs += [msg]
     return msgs
+
+def dataserver_message_decode(msgs, key_generator=None, version=(1,0), **kwargs):
+    '''
+    Convert a list of larpix data server messages into packets. A key generator
+    should be provided if packets are to be used with an ``larpix.io.IO``
+    object. The data server messages provide a ``chip_id`` and ``io_chain`` for
+    keys. Additional keyword arguments can be passed along to the key generator. E.g.::
+
+        from larpix.larpix import Key
+        def ex_key_gen(chip_id, io_chain, io_group):
+            return Key(Key.key_format.format(
+                chip_id=chip_id,
+                io_channel=io_chain,
+                io_group=io_group
+            ))
+
+        msg = b'\\x01\\x00D\\x01\\x00\\x00\\x00\\x00\\x04\\x00\\x00\\x00\\x00\\x00\\x00'
```

Comment:
Other bytestring issue
```suggestion
        msg = b'\x01\x00D\x01\x00\x00\x00\x00\x04\x00\x00\x00\x00\x00\x00'
```